app/scripts/require_config.js

```diff
@@ -21,6 +38,33 @@ require.config({
       name: 'touch-punch'
     }
   ],
+  requireOnDemand: [
+    'fxaClient',
+    'jwcrypto',
+    'passwordcheck'
+  ],
+  // the sriify task will replace sriConfig for production
+  sriConfig: {},
```

Comment:
Add comments before and after this to not modify the line since the build script replaces it.